config/examples/Creality/Ender-3 Pro/CrealityV422/Configuration.h

```diff
@@ -827,14 +826,14 @@
  * *** IT IS HIGHLY RECOMMENDED TO LEAVE THIS OPTION ENABLED! ***
  */
 #define PREVENT_COLD_EXTRUSION
-#define EXTRUDE_MINTEMP 180
+#define EXTRUDE_MINTEMP 170
 
 /**
  * Prevent a single extrusion longer than EXTRUDE_MAXLENGTH.
  * Note: For Bowden Extruders make this large enough to allow load/unload.
  */
 #define PREVENT_LENGTHY_EXTRUDE
-#define EXTRUDE_MAXLENGTH 1000
+#define EXTRUDE_MAXLENGTH 200
```

Comment:
If you use something like Octoprint to help automate filament load/unload or use Marlins built in version then this value needs to be at least 600.  So in this case, I think its V427 config that needs update and not V422.

Reply:
I can make a change to both config files, set it to 600, and maybe include a comment specifying why the value is what it is?